clang/test/Modules/pr73023.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: split-file %s %t
+//
+// RUN: %clang_cc1 -std=c++20 %t/foo.cpp -I%t -fsyntax-only -verify
+
+//--- i.h
+#ifndef FOO_H
```

Comment:
The mismatch between i.h and FOO_H is slightly annoying but don't think it is bad enough to block the change.

Reply:
Done